### PR TITLE
glfwCreateWindow not guarantee create

### PR DESCRIPTION
**Pull request by perminovVS**
Need check glfwCreateWindow and show info, if there is a problem

### CODE REVIEW

**Comment by pandamicro** [comment]
@dabingnn Please check this PR

**Comment by perminovVS** [comment]
error launch application
Example worked it.
Without this, the application would just crash

File: cocos/platform/desktop/CCGLViewImpl-desktop.h [hunk 1]
```diff
@@ -146,6 +146,8 @@ class CC_DLL GLViewImpl : public GLView
     GLFWwindow* _mainWindow;
     GLFWmonitor* _monitor;
 
+    static std::string _glfwError;
```
Comment (dabingnn) [inline cocos/platform/desktop/CCGLViewImpl-desktop.h]: This one should not be `static`.

**Comment by dabingnn** [comment]
@perminovVS Thanks for your contribution.
@pandamicro  Could you merge it?

**Comment by pandamicro** [comment]
Thanks !

**Pull request merged**